packages/@dcl/inspector/src/hooks/sdk/useComponentInput.ts

```diff
@@ -20,14 +20,16 @@ export const useComponentInput = <ComponentValueType extends object, InputType e
   component: Component<ComponentValueType>,
   fromComponentValueToInput: (componentValue: ComponentValueType) => InputType,
   fromInputToComponentValue: (input: InputType) => ComponentValueType,
-  isValidInput: (input: InputType) => boolean = () => true
+  validateInput: (input: InputType) => boolean = () => true,
+  deps: unknown[] = []
```

Comment:
what do you need these `deps` for? They are passed to the `validateInput` callback dependencies, but they are not used by it.

Reply:
they are used by the `useEffect` that triggers the monitor setup. In this specific case, is for re-triggering the validation in case the `files` from the `useFileSystem` hook changes